src/Tizen.Applications.ComponentBased.Port/Tizen.Applications.ComponentBased.Port/ComponentPort.cs

```diff
@@ -78,6 +78,11 @@ public string PortName
         /// <since_tizen> 9 </since_tizen>
         public void AddPrivilege(string privilege)
         {
+            if (privilege == null || privilege.Length == 0)
```

Comment:
```suggestion
            if (string.IsNullOrEmpty(privilege))
```

https://docs.microsoft.com/ko-kr/dotnet/api/system.string.isnullorempty?view=net-5.0

Reply:
Done. Thanks.